acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueMiningAcceptanceTest.java

```diff
@@ -72,4 +72,21 @@ public void shouldStallMiningWhenInsufficientValidators() throws IOException {
     minerNode1.verify(net.awaitPeerCount(0));
     minerNode1.verify(clique.noNewBlockCreated(minerNode1));
   }
+
+  @Test
+  public void shouldStillMineWhenANodeFailsAndHasSufficientValidators() throws IOException {
```

Comment:
`shouldStillMineAfterStoppedValidatorButWithQuroumOfValidators`?

Reply:
i like that. done.